torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1003,6 +1004,33 @@ Operation listAppend(const Node* node) {
   };
 }
 
+template <typename TList>
+Operation listExtend(const Node* node) {
```

Comment:
`Operator()` can take the `std::function<int(Stack& stack)` directly and this doesn't need `node`, so you can declare the op directly (e.g. `int listExtend(Stack& stack) { ... }`)

Reply:
None of the `list` operations seems to need a `node` and uses the same c-tor. Maybe we should do the same to be consistent and follow up with another PR to update all the `list` ops wholesale?

Reply:
Alrightie, created a separate task to track this one.